template.yaml

```diff
@@ -39,23 +42,23 @@ Resources:
             Schedule: rate(1 hour)
             Description: Once an hour
             Enabled: True
+
   FollowerCount:
     Type: AWS::DynamoDB::Table
     Properties:
       AttributeDefinitions:
-        - AttributeName: EpochTime
-          AttributeType: "N"
         - AttributeName: twitter_handle
           AttributeType: "S"
+        - AttributeName: EpochTime
+          AttributeType: "N"
```

Comment:
I love the change--but it would require a destruction / recreation of the existing data, which is sad. There's also categorically no way to transform the existing data and then re-upload it to the table as is, which leaves me conflicted.

Reply:
Backup the existing table first.

Reply:
I suggested creating a new stack (without disrupting the old).  You can then migrate the data by using a script like this:

`
import boto3
dynamodb_resource = boto3.resource('dynamodb')

old_table_name = input('Enter old table name: ')
new_table_name = input('Enter new table name: ')

old_table = dynamodb_resource.Table(old_table_name)
new_table = dynamodb_resource.Table(new_table_name)

# get all items from old table
# got logic from https://stackoverflow.com/questions/36780856/complete-scan-of-dynamodb-with-boto3
response = old_table.scan()
all_items = response['Items']

while 'LastEvaluatedKey' in response:
    response = old_table.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
    all_items.extend(response['Items'])

print(f'The old table has {len(all_items)} items.')

input('press enter when you are ready to insert these items into the new table')

# insert the items into the new table.
with new_table.batch_writer() as batch:
    for item in all_items:
        batch.put_item(
            Item={
                'twitter_handle': item['twitter_handle'],
                'EpochTime': item['EpochTime'],
                'followers': item['followers']
            }
        )
        
        `